app/models/alchemy/page/page_elements.rb

```diff
@@ -199,7 +199,7 @@ def element_definitions_by_name(names)
     #     feed_elements: [element_name, element_2_name]
     #
     def feed_elements
-      elements.named(definition['feed_elements'])
+      elements.published.named(definition['feed_elements'])
```

Comment:
You really want to use `available` scope, because that also [excludes trashed elements](https://github.com/AlchemyCMS/alchemy_cms/blob/master/app/models/alchemy/element.rb#L92)